.editorconfig

```diff
@@ -976,15 +976,8 @@ dotnet_diagnostic.CA2259.severity = suggestion # TODO: Change to warning once us
 
 # We disable the rule category by setting severity to none.
 # Below we enable specific rules by setting severity to warning.
-# Rules are listed below with any options available.
-# Options are commented out if they match the defaults.
 dotnet_analyzer_diagnostic.category-roslynator.severity = none
 
-# A line is too long.
-dotnet_diagnostic.RCS0056.severity = warning
-roslynator_max_line_length = 160 #140
```

Comment:
Please don't undo unrelated changes. Pull changes in with a rebase instead of mangling them with a merge.

Reply:
I pulled the newest bleed an rebased my beaconcursor branch onto that, I have no idea what happened here. This used to work. Let me clean this up.

Reply:
There we go. Better to do this from the console I'm afraid.